onformsubmit-http/utils/auth.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Handles auth for calls to the API
+ * @module Auth
+ */
+const fs = require("fs");
+const google = require("googleapis");
+
+/** Requests and holds tokens */
+class Auth {
+  /**
+   * @param {string} credsFile [relative path to creds file]
+   */
+  constructor(credsFile) {
+    // validate creds file
+    if (fs.existsSync(credsFile)) {
```

Comment:
if you want to use one function in a module you can use es6 module destructuring.
like this const { existsSync } = require("fs");